client/build.gradle

```diff
@@ -72,6 +72,8 @@ dependencies {
 
     implementation 'com.github.mrmike:ok2curl:0.6.0'
 
+    implementation 'com.facebook.stetho:stetho-okhttp3:1.5.1'
```

Comment:
It'd be better to declare dependencies like stetho as `debugImplementation`.

Reply:
The problem with that is that we are adding an extra lib to all apps that is using our client.
We shouldn't use it on the final version, that is because we should use it as `debugImplementation`

Reply:
You do have to divide the code into debug and release versions for `debugImplementation`, but that way the dependency won't be packaged into release builds at all. With the `BuildConfig.DEBUG` check, it'd be inactive since we never init it, but all of its classes are still present in the output artifact. This both makes the apps built on it larger and it a potential security issue, if someone finds a way to activate it in the release version.

Reply:
That is true, the artifact won't be created with the debug flavor.
If we want to use Stetho, it should be included on the sample app of LLC or wait until we merge the three SDK to be able to work with the `debugImplementation` approach